src/osd/PGBackend.cc

```diff
@@ -1270,10 +1270,16 @@ void PGBackend::be_omap_checks(const map<pg_shard_t,ScrubMap*> &maps,
       omap_stats.omap_bytes += obj.object_omap_bytes;
       omap_stats.omap_keys += obj.object_omap_keys;
       if (obj.large_omap_object_found) {
+        pg_t pg;
+        auto osdmap = get_osdmap();
+        osdmap->map_to_pg(k.pool, k.oid.name, k.get_key(), k.nspace, &pg);
+        pg_t mpg = osdmap->raw_pg_to_pg(pg);
         omap_stats.large_omap_objects++;
-        warnstream << "Large omap object found. Object: " << k << " Key count: "
-                   << obj.large_omap_object_key_count << " Size (bytes): "
-                   << obj.large_omap_object_value_size << '\n';
+        warnstream << "Large omap object found. Object: " << k
+                   << " PG: " << pg << " (" << mpg << ")"
```

Comment:
hmm, I think you can simply get_info()->pgid->pgid

Reply:
Yep, I tested with `<< " PG: " << get_parent()->get_info().pgid.pgid <<` and it works fine.

Reply:
I looked at that originally but my thought was to present it in the same way as "ceph osd map" does for consistency and also because I assume we did it that way for a reason. I can change it if necessary.

Reply:
Yep, it looks good to me as @badone explained it is consistent with `ceph osd map` command output which has raw `pg id` also.